timeless/restaurants/table_shapes/views.py

```diff
@@ -49,23 +49,27 @@ def create():
 
 @bp.route("/edit/<int:id>", methods=("GET", "POST"))
 def edit(id):
-    """ Edit table shape with id
-    @todo #162:30min Implement edit() method of TableShape model and
-     update template. Use TableShapeForm for that, see how in works in
-     create function. Check and update html template if it's needed.
-    """
-    if request.method == "POST":
-        flash("Edit not yet implemented")
+    """Edit table shape with id"""
+    table = models.TableShape.query.get(id)
+    if not table:
+        return redirect(url_for("tables.list"))
+
+    form = forms.TableShapeForm(request.form, instance=table)
```

Comment:
@krzyk the form should be read/instantiated inside the ``if``, only if it's a ``POST``, right?

Reply:
@amihaiemil no, user can go to the page with a GET, then it will render a form with filled out fields, in case of POST there is only save to DB from the form